gnomad/assessment/sanity_checks.py

```diff
@@ -104,78 +116,83 @@ def make_filters_sanity_check_expr(
 
 
 def sample_sum_check(
-    ht: hl.Table,
-    prefix: str,
+    t: Union[hl.MatrixTable, hl.Table],
+    subset: str,
     label_groups: Dict[str, List[str]],
-    verbose: bool,
-    subpop: bool = None,
     sort_order: List[str] = SORT_ORDER,
-) -> None:
+    delimiter: str = "-",
+    metric_first_label: bool = True,
+) -> Tuple[dict, dict]:
     """
-    Compute the sum of annotations for a specified group of annotations, and compare to the annotated version.
+    Compute the sum of call stats annotations for a specified group of annotations, compare to the annotated version, and display the result in stdout.
 
-    Results from checking the sum of the specified annotations are printed in the terminal.
+    For example, if pop1 consists of pop1, pop2, and pop3, check that t.info.AC-subset1 == sum(t.info.AC-subset1-pop1, t.info.AC-subset1-pop2, t.info.AC-subset1-pop3).
 
-    :param ht: Table containing annotations to be summed.
-    :param prefix: String indicating sample subset.
-    :param label_groups: Dictionary containing an entry for each label group, where key is the name of the grouping,
-        e.g. "sex" or "pop", and value is a list of all possible values for that grouping (e.g. ["male", "female"] or ["afr", "nfe", "amr"]).
-    :param verbose: If True, show top values of annotations being checked, including checks that pass; if False,
-        show only top values of annotations that fail checks.
-    :param subpop: Subpop abbreviation, supplied only if subpopulations are included in the annotation groups being checked.
+    :param t: Input MatrixTable or Table containing annotations to be summed.
+    :param subset: String indicating sample subset.
+    :param label_groups: Dictionary containing an entry for each label group, where key is the name of the grouping, e.g. "sex" or "pop", and value is a list of all possible values for that grouping (e.g. ["XY", "XX"] or ["afr", "nfe", "amr"]).
     :param sort_order: List containing order to sort label group combinations. Default is SORT_ORDER.
-    :return: None
+    :param delimiter: String to use as delimiter when making group label combinations.
+    :param metric_first_label: If True, metric precedes label group, e.g. AC-afr-male. If False, label group precedes metric, afr-male-AC.
+    :return: Tuple of dictionaries
```

Comment:
PART OF LARGE SUGGESTION
```suggestion
    :return: Dictionary of sample sum field check expressions and display fields
```